### DISCUSSION TITLE
How should Back to top information/guidelines be captured within the DOC site?

### DISCUSSION THREAD

**Discussion by bennettcc** [NCIOCPL/ncids#1472] [NCIDS Launch]
During Paul K's completion of this story below, he noticed that back to top isn't documented clearly within the Footer doc site page (https://designsystem-dev.cancer.gov/develop/components/footer).

https://github.com/NCIOCPL/ncids/issues/1310

Questions to be discussed:
- Should back to top be its own page within the documentation site? Or should it continue to live within the footer page?
- If living on the footer page what guidance can be added to provide more information about the back to top behavior, visual needs, etc (BTT is included within the code snippets but there is no mention of it throughout the guidance)
- Should the overview image be updated to include the back to top functionality example?

**Comment by sarinapadilla** [comment]
We updated the Footer page with #1529:
* Back to Top is now in the overview image
* The footer has JS enabled, so it includes the Back to Top in the preview
